xmodule/services.py

```diff
@@ -222,3 +225,98 @@ def rebind_noauth_module_to_user(self, block, real_user):
         # now bind the module to the new ModuleSystem instance and vice-versa
         block.runtime = inner_system
         inner_system.xmodule_instance = block
+
+
+class PublishService(Service):
```

Comment:
Could we rename this to something like `EventPublishingService` or something like that? The name "PublishService" could be mistaken for something related to publishing content, as opposed to events.

Reply:
Good idea! Done.